### DISCUSSION TITLE
Discrepancy in Variant Pair and Misorientation Angle Data

### DISCUSSION THREAD

**Discussion by bamobe** [mtex-toolbox/mtex#2183] [Ask Anything]
Hello everyone,

I've been working on plotting variant pairs with V1 and the misorientation angles between the BCC phases in steel. I've attached my plots here.

However, I have encountered a puzzling discrepancy: in the variant pair plot, the V1-V4 pair, which corresponds to a low-angle boundary, shows a high fraction. Conversely, the fraction of low-angle boundaries in the misorientation angle plot is notably small.

I would appreciate your insights on why there might be such a difference in the representation of low-angle boundaries between these two analyses.
Also, I've attached the commands that i used for my EBSD data.

Thank you for your time and assistance.
![Misorientation](https://github.com/user-attachments/assets/9e371109-6091-472e-beeb-065fc741371f)
![Equivalant Variant Pairs](https://github.com/user-attachments/assets/a43878a1-a210-41ae-9bc9-214c74ea99e1)
![Variant Pairs with V1](https://github.com/user-attachments/assets/a347c582-d404-4f54-9b70-146f5b420cf6)

```matlab
ebsd = EBSD.load('950  - Iso 400.ctf')
[grains,ebsd.grainId] = calcGrains(ebsd('indexed'),'angle',3*degree);
ebsd(grains(grains.grainSize < 3)) = [];
[grains,ebsd.grainId] = calcGrains(ebsd('indexed'),'angle',3*degree);
grains = smooth(grains,5)

%%% This line is responsible for plotting misorientation angle
gB = grains.boundary('AlphaP','AlphaP')
plotAngleDistribution(gB.misorientation)
%%%

job = setParentGrainReconstructor(ebsd,grains,Ini.cifPath);
KS = orientation.KurdjumovSachs(job.csParent,job.csChild);
job.p2c = KS;
job.calcParent2Child;
job.calcVariantGraph('threshold',4*degree,'tolerance',3.5*degree);
job.clusterVariantGraph('includeSimilar');
job.calcParentFromVote('minProb',0.5)
job.calcGBVotes('p2c','reconsiderAll')
job.calcParentFromVote
job.mergeSimilar('threshold',7.5*degree);
job.mergeInclusions('maxSize',150);
job.calcVariants;

%%%This line is responsible for plotting variant pairs with equivalant Id
variantBoundaries_map = plotMap_variantPairs(job,'linewidth',1.5);
plotMap_variantPairs(job,'linewidth',1.5)
%%%

%%%This line is responsible for plotting variant pairs with V1
[newGrains,~] = computeVariantGrains(job);
newGrains = newGrains(job.csChild);
vGroupIds = {[1 2],...
    [1 3; 1 5],...
    [1 4],...
    [1 6],...
    [1 7],...
    [1 8],...
    [1 9; 1 19],...
    [1 10; 1 14],...
    [1 11; 1 13],...
    [1 12; 1 20],...
    [1 15; 1 23],...
    [1 16],...
    [1 17],...
    [1 18; 1 22],...
    [1 21],...
    [1 24]};
	
out1 = computeGrainPairs(newGrains,'variant','group',vGroupIds,'plot');
```

**Comment by frankNiessen** [comment]
Dear Bamobe, 

thank you for sharing this. The problem you describe is in fact related to the ORTools toolbox, so I would appreciate if you could move/rewrite your request here: https://github.com/ORTools4MTEX/ORTools/discussions 
If you could share the data or a representative subset of it, chances will be that we can better reproduce your problem. 

Best wishes
Frank
